website/source/partials/builders/_boot-command.html.md

```diff
@@ -0,0 +1,68 @@
+There are a set of special keys available. If these are in your boot
+command, they will be replaced by the proper key:
+
+-   `<bs>` - Backspace
+
+-   `<del>` - Delete
+
+-   `<enter> <return>` - Simulates an actual "enter" or "return" keypress.
+
+-   `<esc>` - Simulates pressing the escape key.
+
+-   `<tab>` - Simulates pressing the tab key.
+
+-   `<f1> - <f12>` - Simulates pressing a function key.
+
+-   `<up> <down> <left> <right>` - Simulates pressing an arrow key.
+
+-   `<spacebar>` - Simulates pressing the spacebar.
+
+-   `<insert>` - Simulates pressing the insert key.
+
+-   `<home> <end>` - Simulates pressing the home and end keys.
+
+-   `<pageUp> <pageDown>` - Simulates pressing the page up and page down keys.
+
+-   `<leftAlt> <rightAlt>` - Simulates pressing the alt key.
+
+-   `<leftCtrl> <rightCtrl>` - Simulates pressing the ctrl key.
+
+-   `<leftShift> <rightShift>` - Simulates pressing the shift key.
+
+-   `<leftSuper> <rightSuper>` - Simulates pressing the ⌘ or Windows key.
+
+-   `<wait> <wait5> <wait10>` - Adds a 1, 5 or 10 second pause before
+    sending any additional keys. This is useful if you have to generally wait
+    for the UI to update before typing more.
+
+-   `<waitXX>` - Add an arbitrary pause before sending any additional keys. The
+    format of `XX` is a possibly signed sequence of decimal numbers, each with
+    optional fraction and a unit suffix, such as `300ms`, `-1.5h` or `2h45m`.
```

Comment:
What does `<wait-1.5h>` do?

Reply:
copied and pasted this from the go docs and missed that. for sure we should only accept positive values.